src/utils/systemIntake.ts

```diff
@@ -7,6 +7,11 @@ import {
   RequestType
 } from 'types/systemIntake';
 
+// NOTE: these utility functions take strings rather than more restricted types so they can operate on values coming from GraphQL queries,
```

Comment:
@ClayBenson94 Is this comment clear and understandable?

Reply:
I like this. And it's a good catch as well!